source/components/CouldDo/_tests/CouldDoContainer.test.js

```diff
@@ -7,23 +8,63 @@ import CouldDoContainer from '../CouldDoContainer'
 import globalState from '../../utilities/globalState'
 
 describe( ' <CouldDoContainer />', () => {
-  let wrapper
-
-  beforeEach( () => {
-    globalState.set( mockGlobalState )
-    wrapper = mount(
-      <MemoryRouter>
-        <CouldDoContainer />
-      </MemoryRouter>
-    )
+  let wrapper, globalStateSetSpy
+
+  context( 'when there is a global state', () => {
+
+    beforeEach( () => {
+      globalState.set( mockGlobalState )
+      globalStateSetSpy = sinon.spy( globalState, 'set' )
+      wrapper = mount(
+        <MemoryRouter>
+          <CouldDoContainer />
+        </MemoryRouter>
+      )
+    })
+
+    afterEach( () => {
+      wrapper.unmount()
+      globalStateSetSpy.restore()
+    })
+
+    it( 'checks logic of position prop', () => {
+      expect( wrapper.find( 'CouldDoContainer' ).nodes[0].findPosition( 0, 5 ) ).to.equal( 'beginning' )
+      expect( wrapper.find( 'CouldDoContainer' ).nodes[0].findPosition( 4, 5 ) ).to.equal( 'end' )
+      expect( wrapper.find( 'CouldDoContainer' ).nodes[0].findPosition( 3, 5 ) ).to.equal( 'middle' )
+    })
+
+    it( 'renders CouldDo component', () => {
+      expect( wrapper.find( 'CouldDo' ).length ).to.eql( 1 )
+    })
+
+    it( 'should change currentCouldDoIndex to 1 when down button is clicked', () => {
```

Comment:
Since we are not testing the button click here, lets change this description to something like 'calling nextCouldDo() increases currentCouldDoIndex by 1'. Add an expect before calling nextCouldDo() that checks that currentCouldDoIndex is 0.